### PR TITLE
Add module Grad

### PR DESCRIPTION
**Pull request by sathish-a**
With reference to #40. Added a module `Grad`. `Grad` internally calls `GradValues` and return only the gradients.
* No linter error.
* All tests passed.

### CODE REVIEW

File: objax/gradient.py [hunk 1]
```diff
@@ -71,3 +71,28 @@ def __call__(self, *args):
                                        list(args))
         self.vc.subset(BaseState).assign(changes)
         return g, outputs
+
+
+class Grad(ModuleWrapper):
```
Comment (david-berthelot) [inline objax/gradient.py:76]: Can you move the new function below GradValues (we try to keep the order alphabetical within files)?

File: objax/gradient.py [hunk 2]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__all__ = ['GradValues']
+__all__ = ['GradValues', 'Grad']
```
Comment (david-berthelot) [inline objax/gradient.py:15]: Suggestion `Grad` followed by `GradValues` (see next comment).

**Review by david-berthelot — CHANGES_REQUESTED** [review]
Can you also and entry to https://github.com/google/objax/blob/master/docs/source/objax/objax.rst
for the API to show in docs?

Thanks!

File: objax/gradient.py (hunk 1 above)
Comment (sathish-a) [reply objax/gradient.py:76]: Sure. This applies to the test script also right?

**Review by david-berthelot — APPROVED** [review]
